README.md

```diff
@@ -5,8 +5,16 @@ A system for discovering and tracing the composition of shipped products.
 ## Development
 
 To setup a development environment:
-* Create and activate a [Python virtual environment](https://virtualenv.pypa.io/en/stable/)
-    (Python 3 is preferred)
+* Create and activate a [Python virtual environment](https://docs.python.org/3/library/venv.html)
+    (Python 3.6 or later is required):
+  ```bash
+  $ python3 -m venv --system-site-packages venv
```

Comment:
I don't think we want to use `--system-site-packages` since it can lead to "works on my machine" issues.

Reply:
How else do you get the `rpm` module into your venv then?

Reply:
Does `pip install rpm` provide a different dependency than you were expecting?

Reply:
That package is just a stub on [PyPI](https://pypi.org/project/rpm/):

```python
>>> import rpm
/home/.../venv/lib64/python3.6/site-packages/rpm.py:15: UserWarning: The RPM Python bindings are not currently available via PyPI.

Please install them with your distro package manager (typically called
'python2-rpm' or 'python3-rpm'), and ensure that any virtual environments
needing the API are configured to be able to see the system site packages
directory.
```

Reply:
Okay, you should mention that you need to install the RPM then before creating the virtualenv.

Reply:
It's installed by default on all RPM-based systems (dnf/yum require it as a dependency). The point here is that if you create an empty venv, it won't be available unless you allow that venv access to the system packages. I don't think it's possible to specify that you only want to pull in the `rpm` module but not anything else from the system unless you do some manual symlinking.